pkg/sql/codegen/pai/codegen.go

```diff
@@ -220,12 +222,26 @@ func Train(ir *ir.TrainStmt, session *pb.Session, modelName, cwd string) (string
 	if err != nil {
 		return "", err
 	}
+
+	if strings.HasPrefix(strings.ToLower(ir.Estimator), "xgboost") {
+		// run xgboost on PAI platform
```

Comment:
Should we encapsulate L227 ~ L236 into a function named `XGBoostTrainAndSave`, like that on L239, there is `TFTrainAndSave` (better named TensorFlowTrainAndSave`?)